src/runtime/button-api.js

```diff
@@ -23,8 +23,8 @@ const TITLE_LANG_MAP = {
   'ar': 'الاشتراك عبر Google',
   'de': 'Abonnieren mit Google',
   'es': 'Suscríbete con Google',
-  'es-latam': 'Suscribirse con Google',
-  'es-latn': 'Suscribirse con Google',
+  'es-latam': 'Suscríbete con Google',
```

Comment:
This will only change the button title, it won't change the text on the button which comes from the SVG.

Reply:
The SVG contains correct text. The title was not matching with the text.